NamespaceAndPackage.md

```diff
@@ -1,8 +1,7 @@
 # Namespace and Package names
 
 ## Introduction
-This document describes the key package names (namespaces, etc.). This document uses terminology
-(MUST, SHOULD, etc) from [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt).
+This document describes the key package names (namespaces, etc.).
 
 ## Structure
 The top level package name MUST be **opencensus** (for URL model MUST use **io.opencensus**).
```

Comment:
Does this mean we need go.opencensus.io or alike as a Go import path?

Reply:
Sounds about right. we should chat about this because I am not sure how this go distribution of packages works.

Reply:
More natural to go with `github.com/census-instrumentation/trace`, `github.com/census-instrumentation/stats` isn't it? Or I guess `opencensus.io/trace`?

Reply:
Filed an issue for this:
https://github.com/census-instrumentation/opencensus-specs/issues/15